schema/osquery_fleet_schema.json

```diff
@@ -17754,6 +17754,12 @@
         "type": "text",
         "required": false,
         "description": "The Update Framework update channel used for the Fleet Desktop executable."
+      },
+      {
+        "name": "uptime",
+        "type": "bigint",
+        "required": false,
+        "description": "Uptime of the orbit process in seconds."
```

Comment:
I don't see this in the requirements. Did you add it on the fly as nice to have?